app/containers/AccountProvider/saga.js

```diff
@@ -176,6 +233,10 @@ export function* updateAccWorker({ eos }) {
     }
 
     if (account) {
+      if (profileInfo.pay_out_rating > 35) {
```

Comment:
35 needs to be moved to constants file on the same container

Reply:
Moved